### ISSUE TITLE
Audit Ares for accessibility

### ISSUE THREAD

**Issue by kevinreiss** [pulibrary/dacs_handbook#331]
- [x] Use the [accessibility audit procedure](https://github.com/pulibrary/dacs_handbook/blob/main/Accessibility/audit_procedure.md) to audit using the course menu and adding an item when your account is in "instructor" mode. Note: the Item Tools choices have been flagged by John Jameson as being non-navigable via keyboard. 
- [x] Use the [accessibility audit procedure](https://github.com/pulibrary/dacs_handbook/blob/main/Accessibility/audit_procedure.md) to audit browsing the readings list and viewing an individual item when your account is in "student" mode
- [x] We create an issue describing a known keyboard navigation issue for the Item Tools menu. See notes below. 
- [x] Use the accessibility issue template to create github issues for any issues that you encounter.

## Accessing Ares
ARES is only used within [canvas](https://canvas.princeton.edu/) courses. To access a course with active ARES reserve items to test you need to login via SSO. 

1. ~~Make sure your account is added as an instructor to the "Library Sandbox Course" in both canvas and ARES~~ - Everyone's accounts have been added.
2. Sign in to canvas.
3. Select the "Library Sandbox" Course. 
4. Select the option "reserves" in the course menu. 
5. Once on this tab you can toggle between student and instructor modes.  

## Notes
The code for our ARES instance is hosted by atlas [here](https://github.com/AtlasSystems/hosting-ares-princeton). 

A known accessibility problem is the lack of keyboard navigation on the Item Tools Menu.

<img width="339" height="253" alt="Image" src="https://github.com/user-attachments/assets/dd05832a-9dec-4a0d-8047-3a773e84ccee" />

**Comment by rladdusaw** [comment]
Instructor report: https://docs.google.com/spreadsheets/d/1faMQrAjG8mbZdvVjOJ9Gd9KGDIlo7xyezN7hOrS3B9A/edit?gid=0#gid=0
Student report: https://docs.google.com/spreadsheets/d/1dUUT4WjDU9ck7N1x97RpYMzbLWye714x6w22yoYfKlQ/edit?gid=0#gid=0

Will create tickets tomorrow.

**Comment by rladdusaw** [comment]
Audit complete and all issues are created. Closing as completed.
